### ISSUE TITLE
[SF][A11y][0.26.0][MessageBox]modal dialog announcement of MessageBox is missing

### ISSUE THREAD

**Issue by ksblklu** [SAP/ui5-webcomponents-react#3431]
**Describe the bug**
Modal dialog announcement is missing after opening MessageBox Component

**To Reproduce**
1. Go to [MessageBox Document Page](https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-messagebox--default-story)
2. Click on 'open MessageBox' button
3. Inspect speech history of JAWS screen reader
4. See announcement of MessageBox modal dialog is missing

<img width="1707" alt="image" src="https://user-images.githubusercontent.com/27538066/177506399-eb1ff6d9-5c45-4c51-b124-7069dc1cd520.png">

**Expected behavior**
Modal dialog message should be announced after the MessageBox opens, same as dialog:
<img width="1745" alt="image" src="https://user-images.githubusercontent.com/27538066/177506802-07b2f279-8266-4329-a896-c3aa858c8fee.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.5.0
`@ui5/webcomponents-react` version: 0.26.0
Operating System: Windows
Browser: Chrome

**Comment by don-obrien** [comment]
@MarcusNotheis , When will this fix be released?   We need to consume it by Sept. 15th.

**Comment by MarcusNotheis** [comment]
We're currently blocked by https://github.com/SAP/ui5-webcomponents/issues/5778, so this is unfortunately out of our hands.
As soon as there is a 1.7.1 version available, we'll release our new version.

**Comment by ksblklu** [comment]
Hi @MarcusNotheis , this issue still exist in 0.28.1, would you please have a look?
<img width="1697" alt="image" src="https://user-images.githubusercontent.com/27538066/190090785-9fcbfc29-78e7-408a-9e3e-76391ea0ddf7.png">

**Comment by MarcusNotheis** [comment]
Hey @ksblklu,
does this issue only occur for the MessageBoxType `Confirm`? Or for other types like `Success`, `Error`, etc. as well?

**Comment by ksblklu** [comment]
hi @MarcusNotheis ,  I've tested Messagebox with all MessageBoxTypes, and none of it made correct announcement "Modal dialog" after opening. Please kindly reopen this issue and take a look, thanks.

**Comment by don-obrien** [comment]
@MarcusNotheis , can you remove the author action label, if you are satisfied with the response?

**Comment by MarcusNotheis** [comment]
Issue is blocked by https://github.com/SAP/ui5-webcomponents/issues/5796

**Comment by MarcusNotheis** [comment]
Yes, we'll now starting working on it again
